srcpkgs/git-credential-oauth/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'git-credential-oauth'
+pkgname=git-credential-oauth
+version=0.8.0
+revision=1
+build_style=go
+go_import_path=github.com/hickford/git-credential-oauth
+short_desc="Git credential helper that authenticate using OAuth"
+maintainer="M Hickford <mirth.hickford@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/hickford/git-credential-oauth"
+distfiles="https://github.com/hickford/git-credential-oauth/archive/v${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://github.com/hickford/git-credential-oauth/archive/refs/tags/v${version}.tar.gz"
```

Reply:
Both links work, is there any difference?

Reply:
Just a good practice, and there have been cases where a tag is named the same as a branch and fetch fails due to ambiguity